airflow/providers/google/suite/transfers/gcs_to_gdrive.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         source_bucket: str,
         source_object: str,
         destination_object: str | None = None,
-        destination_folder_id: str | None = None,
+        destination_folder_id: str = "root",
```

Comment:
why do we change the default?

Reply:
> changed the default value for the attribute `GCSToGoogleDriveOperator.destination_folder_id` for making it consistent with `GoogleDriveHook.folder_id = "root"`. Previously the operator was passing the `None` value to the hook that expects only strings, which caused errors.

I think this is the reason (I also had the question when reviewing this PR)

Reply:
Yes, @Lee-W  is right, thank you.

Reply:
It looks like a breaking change is it not?

Reply:
Look like `destination_folder_id` getting used in calling `upload_file` of GoogleDriveHook which has a default value for this param is `root` so it should not break existing user code. https://github.com/apache/airflow/blob/main/airflow/providers/google/suite/hooks/drive.py#L263

Reply:
Indeed. [API documentation](https://developers.google.com/drive/api/guides/folder) claims that 
```
you can use the alias root to refer to the root folder anywhere a file ID is provided.
```
So it is a good default value for the folder id and it is already used in the [hook](https://github.com/apache/airflow/blob/main/airflow/providers/google/suite/hooks/drive.py#L263). But the operator's default value is inconsistent and actually causes errors: the execution comes to [this method](https://github.com/apache/airflow/blob/main/airflow/providers/google/suite/hooks/drive.py#L163) which fails when it passes `None` to the [API call](https://developers.google.com/drive/api/reference/rest/v3/files/get).